packages/components/organisms/f-form/src/components/_tests/Form.test.js

```diff
@@ -1,10 +1,92 @@
 import { shallowMount } from '@vue/test-utils';
-import VForm from '../Form.vue';
+import FForm from '../Form.vue';
+
+const formData = {
+    formFields: {
```

Comment:
Did you try using an array instead of an object? I think it would make a bit more sense to me; the ordering is a bit more explicit and you wouldn't need to name each object on top of its `name` property.